cmd/ui/src/components/GraphEvents.tsx

```diff
@@ -181,104 +161,80 @@ export const GraphEvents = forwardRef(function GraphEvents(
             },
             mousemove: (event) => {
                 if (draggedNode) {
-                    // Prevent sigma to move camera:
-                    event.preventSigmaDefault();
-                    event.original.preventDefault();
-                    event.original.stopPropagation();
+                    // Prevent Sigma from moving camera
+                    preventAllDefaults(event);
                 }
             },
             doubleClickNode: (event) => {
-                if (!onDoubleClickNode) return;
```

Comment:
There was never a doubleClick handler passed in, so this extra logic could be removed.